hook/DestIPFoundHook.js

```diff
@@ -306,6 +348,7 @@ class DestIPFoundHook extends Hook {
       if (!this.isFirewalla(domain)) {
         try {
           cloudIntelInfo = await intelTool.checkIntelFromCloud(ip, domain, fd);
```

Comment:
should check domain cache here

cache hit => return the data in the cache instead of cloud
cache miss => check cloud => update cache => return